mod/metrics/set/abstract/metric_answer/views.rb

```diff
@@ -24,14 +31,24 @@
     ]
   end
 
+  def metric_designer_card
+    Card.fetch metric_card.metric_designer
```

Comment:
I would be surprised if `metric_card` doesn't have a `metric_designer_card` method.

Reply:
I am not sure if that method exists? How would I know if a method exists or not? any idea? unless by guessing and trying it on pry

Reply:
I think you have two possibilities a) try to call it or b) try to find the definition
the easiest way for a) is open a console (`be decko c`), fetch a metric card and call `metric_designer_card`
for b) you have to search for card accessor `:metric_designer` (it defines also a card method) and method definition `metric_designer_card`. 
   If one of both is defined in a `type/metric` set you are fine

also possible but only only a sufficient and not a necessary condition: find all `metric_designer_card` calls and look for a call on a metric card object

Reply:
I'll handle this.